mender-deb-package

```diff
@@ -196,44 +200,76 @@ build_packages() {
     dh_make --file /orig/"${DEB_PACKAGE}_${DEB_VERSION_NO_REV}".orig.tar.gz --packagename "${DEB_PACKAGE}_${DEB_VERSION_NO_REV}" -s -y || true
   fi
 
+  dpkg_build_ignore_flag="--diff-ignore=(.git|vendor)"
+
+
+  skip_patches_flag="--source-option=--skip-patches"
+  if [ "$OS_DISTRO" = "debian" -a "$OS_CODENAME" = "buster" ]; then
+    echo "deb http://deb.debian.org/debian buster-backports main" > /etc/apt/sources.list.d/buster-backports_deps.list
+    cat <<EOF> /etc/apt/preferences.d/prefer_backports.pref
+Package: cmake*
+Pin: release a=buster-backports
+Pin-Priority: 999
+EOF
+    apt update
+    # We want the patches applied for Debian Buster
+    rm -f debian/source/local-options
+    skip_patches_flag=
+  else
+    # Skip applying the patches
+    rm -f debian/patches/series
+  fi
+
+  # Install the builder architecture independent dependencies from the debian/control file
+  mk-build-deps --install --build-indep --tool='apt-get -o Debug::pkgProblemResolver=yes --no-install-recommends --yes' debian/control
```

Comment:
We should probably discuss what we want to do with this?

Reply:
This is way above my head (I hope Lluis understands this! 🙂)

Reply:
We should skip this on debian/armhf build, where we have tracked down all the dependencies (in [here](https://github.com/mendersoftware/mender-test-containers/pull/361/files#diff-8843be8793b8fa8003b51d477dfa6943c2ea7ac81a8cddb4e327c59f98dbd07dR14)).

For the rest we can leave it (which is a better approach than having the dependencies installed in the Dockerfile before hand)